src/cftlib/java/uk/gov/hmcts/reform/sscs/CftlibConfig.java

```diff
@@ -41,6 +41,9 @@ public void configure(CFTLib lib) throws Exception {
         lib.createIdamUser("sscs-citizen2@hmcts.net",
             "citizen"
         );
+        lib.createIdamUser("sscs-citizen-khan@hmcts.net",
```

Comment:
how comes we need this new user?

Reply:
we don't NEED a new user. this is just for local testing, as I was using the same user over and over again, I wanted another user so that I can see different cases in MYA